tests/all_mode/run.sh

```diff
@@ -32,6 +32,9 @@ function run() {
 
     # use sync_diff_inspector to check data now!
     check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
+
+    check_metric $WORKER1_PORT 'dm_syncer_replication_lag{task="test"}' 0 1
+    check_metric $WORKER2_PORT 'dm_syncer_replication_lag{task="test"}' 0 1
```

Comment:
check before the task started?
whether needing to check after replicated some binlog events.

Reply:
This metric is task related, if task has not started, there is no `dm_syncer_replication_lag` for this task, and if task is not in syncer unit, the `dm_syncer_replication_lag` value is the updated timestamp.
Do you mean check metric does not exist before task is started?

Reply:
Are these two lines (L36, L37) checking metrics but the task is not started yet?

I mean, check metric after some data are replicated?

Reply:
the task starts at L25, L28 ensures load stage is done, L30,L31 sends increment payload in upstream.

Reply:
Oh, my mistake.